docs/keda-development.md

```diff
@@ -0,0 +1,76 @@
+##  Development
+
+###  Project structure
+
+Keda Manager codebase is scaffolded with `kubebuilder`. For more information on `kubebuilder`, visit the [project site](https://github.com/kubernetes-sigs/kubebuilder).
+
+- `config`: A directory containing the [kustomize](https://github.com/kubernetes-sigs/kustomize) YAML definitions of the module. For more information, see [kubebuilder's documentation on launch configuration](https://book.kubebuilder.io/cronjob-tutorial/basic-project.html#launch-configuration).
+- `api`: Packages containing Keda CustomResourceDefinitions (CRD). 
+- `controllers`: Package containing the implementation of the module's reconciliation loop responsible for managing Keda custom resources (CRs).
+- `Dockerfile`: The definition of the `keda-manager-module` image.
+- `bin`: A directory with binaries that are used to build/run project.
+- `config.yaml`: Configuration file to override module's Helm chart properties.
+- `docs`: Contains context documentation for the project.
+- `hack`: A directory containing scripts and makefiles that enhance the root `Makefile` capabilities.
+- `pkg`: Contains packages used in the project.
+- `keda.yaml`: Kubernetes objects that represent `keda module`.
+
+
+### Prerequisites
+
+- Access to a Kubernetes cluster
+- [Go](https://go.dev/)
+- [k3d](https://k3d.io/v5.4.6/)
+- [Docker](https://www.docker.com/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [kubebuilder](https://book.kubebuilder.io/)
+
+### Useful Make targets 
+
+You can build and run the Keda Manager in the Kubernetes cluster without Kyma.
+For the day-to-day development on your machine, you don't always need to have it controlled by Kyma's Lifecycle Manager.
```

Comment:
Can we explain in which cases it must/should be controlled by Lifecycle Manager? 
"you don't always need to have it for day-to-day development" is pretty vague and I'm not sure it's enough guidance for readers to decide whether they need it.